rio_tiler/io/__init__.py

```diff
@@ -1,5 +1,8 @@
 """rio-tiler.io"""
 
 from .base import BaseReader, MultiBandReader, MultiBaseReader  # noqa
-from .cogeo import COGReader  # noqa
+from .rasterio import Reader
```

Comment:
In preparation for merging #530, it makes sense to me to rename `cogeo` to `rasterio`